fvm/src/kernel/error.rs

```diff
@@ -26,7 +28,7 @@ macro_rules! syscall_error {
     };
 }
 
-// NOTE: this intentionally does not implemnent error so we can make the context impl work out
+// NOTE: this intentionally does not implement error so we can make the context impl work out
 // below.
 #[derive(Display, Debug)]
```

Comment:
Is it not possible to derive `Clone`?

Reply:
No, because anyhow::Error doesn't feel inspired to be cloned.